panel/chat/langchain.py

```diff
@@ -68,12 +69,13 @@ def _update_active(self, avatar: str, label: str):
             self._active_user = f"{self._active_user} - {label}"
 
     def _stream(self, message: str):
-        return self.instance.stream(
-            message,
-            user=self._active_user,
-            avatar=self._active_avatar,
-            message=self._message,
-        )
+        if message.strip():
+            return self.instance.stream(
+                message,
+                user=self._active_user,
+                avatar=self._active_avatar,
+                message=self._message,
+            )
 
```

Comment:
Should this return self._message?

Reply:
I would not know. All I can say is that in the different examples I have tried this works.

Reply:
Please propose what you see fit or resolve the conversation. Thanks.

Reply:
I created https://github.com/holoviz/panel/pull/5718/

Reply:
Thx